tests/wdio/hideInputCaret-spec.js

```diff
@@ -0,0 +1,20 @@
+describe('hideInputCaret', () => {
+  before(() => {
+    browser.url('/input.html');
+    // The input explicitly has the caret-color set to black
+    Terra.hideInputCaret('#inputID');
+  });
+
+  it('validates the input\'s caret-color is transparent', () => {
+    const element = browser.element('#inputID');
+    // wdio parses colors with rgb2hex so it becomes a rgba value
+    // http://v4.webdriver.io/api/property/getCssProperty.html
+    expect(element.getCssProperty('caretColor').value).to.equal('rgba(0,0,0,0)');
+  });
+
+  it('validates the textarea\'s caret-color is transparent', () => {
+    const element = browser.element('#textareaID');
+    // Terra service automatically sets caretColor at root level to transparent
+    expect(element.getCssProperty('caretColor').value).to.equal('rgba(0,0,0,0)');
+  });
+});
```

Comment:
We need to add a couple things here (I think):

1. We need a test that isn't providing a custom selector to validate the default 'body' selector is working as expected

2. Do we need to click inside the input elements to trigger the blinking cursor?

Reply:
@Blackop778 thoughts?

Reply:
1. Done in 368a5aa21c202fa551437dd994ee11a4d65bb46c if I am interpreting correctly
2. The `caret-color` property is set regardless of whether the element is focused or not. I tested in chrome and firefox by checking the computed style without the input clicked.

Reply:
@Blackop778
I think you are right, these use cases are being hit. They are a little confusing though; let's break these up a bit:

```
describe('hideInputCaret', () => { 
    before(() => browser.url('/input.html'));

    it('validates caret is transparent for body by default' ()=>{});
    it('validates the textarea caret color gets inherited as transparent' ()=>{});
    it('validates the input caret color gets inherited as transparent' ()=>{});

    it('sets input caret color', () => {
        // set color
        // validate color
    });

    it('successfully hides caret by setting color to transparent', () => {});

    it('throws error for a non-existent element', () => {});
});


```

Reply:
Reorganized in e56a2ef1c493660b497bb9277b3d6ad06eca668e